api/python/cell_census/src/cell_census/release_directory.py

```diff
@@ -31,7 +31,34 @@
 
 
 def get_release_description(tag: str) -> CensusReleaseDescription:
-    """Get release description for given tag. Raises KeyError if unknown tag value."""
+    """
+    Get release description for given tag from the Cell Census release directory.
+    Raises KeyError if unknown tag value.
+
+    Parameters
+    ----------
+    tag : str
```

Comment:
Is `tag` different than `census_version`? if not, maybe rename to the latter for consistency.  (Also, seeing that we use the term "release" in this method `CensusRelease*` types, wondering if `version` and `release` should standardize on one term.)

Reply:
yeah, I don't like any of the names, hence the thrashing :-).   Will fix.